src/js/components/picklist/index.js

```diff
@@ -190,7 +190,7 @@ const Picklists = CollectionView.extend({
     this.addChildView(picklist);
   },
   viewFilter(childView) {
-    return !!childView.children.length;
+    return !!size(childView.children) || childView.shouldShow;
```

Comment:
this previously required `children` assuming all views were collection views on a picklist.  This allows a view without children to pass it's view-ability to a flag on the view